sandbox/src/mocks/R4-OperationOutcome.json

```diff
@@ -0,0 +1,23 @@
+{
+  "meta": {
+    "profile": [
+      "https://www.hl7.org/fhir/R4/operationoutcome.html"
+    ]
+  },
+  "resourceType": "OperationOutcome",
+  "issue": [
+    {
+      "severity": "error",
+      "code": "invalid",
+      "diagnostics": "Service Ids not in a valid format and the valid format should be whole numbers i.e 123450",
```

Comment:
I'd prefer if the example error for R4 would be more generic. This will look unrealistic on endpoints that do not deal with service at all. We could just reuse the same error message that we had for STU3.